commons/clever-tools/ssh-access.md

```diff
@@ -25,11 +25,11 @@ While direct SSH access to instances is not recommended in an [immutable infrast
 
 <div class="panel panel-warning">
   <div class="panel-heading">
-    <h4 class="panel-title">SSH access is disabled on Docker instances</h4>
+    <h4 class="panel-title">SSH on Docker instances</h4>
   </div>
   <div class="panel-body">
     <p>
-    For security reasons, SSH access is disabled for Docker applications. Contact the support if you need help to debug your Docker applications.
+      SSH on docker instances will redirect you inside the running container. You must have `/bin/bash` installed (usually from the `bash` package) otherwise the SSH connection will fail.
```

Comment:
s/docker/Docker
Should we use the attach terminology or whatever is used by Docker insteaf of redirect inside or connection?

Reply:
Yes, I guess I could

Reply:
Even though in that case attach isn't really what we do: `attach:      Attach local standard input, output, and error streams to a running container`

What we do is `docker exec -it <container> bash`.